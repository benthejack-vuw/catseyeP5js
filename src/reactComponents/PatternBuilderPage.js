import React from 'react'

import PatternOptionsPanel from './PatternOptionsPanel'
import GridSelectionPanel  from './GridSelectionPanel'
import PatternDisplay from './PatternDisplay.js'

const PatternBuilderPage = () => (
    <div>
        <PatternDisplay/>
        <PatternOptionsPanel/>
        <GridSelectionPanel/>
    </div>
)

export default PatternBuilderPage;