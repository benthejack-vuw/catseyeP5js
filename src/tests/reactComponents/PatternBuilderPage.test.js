import React from 'react';
import { shallow } from 'enzyme';
import PatternBuilderPage from '../../reactComponents/PatternBuilderPage';

test('should render PatternBuilderPage correctly', ()=>{
    const wrapper = shallow(<PatternBuilderPage startLogout={()=>{}}/>);
    expect(wrapper).toMatchSnapshot();
})