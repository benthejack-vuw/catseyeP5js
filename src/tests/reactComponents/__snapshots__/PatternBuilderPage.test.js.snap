// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PatternBuilderPage correctly 1`] = `
<div>
  <PatternDisplay />
  <Connect(PatternOptionsPanel) />
  <GridSelectionPanel />
</div>
`;
